src/test/java/model/ProductIntegrationTest.java

```diff
@@ -36,7 +36,7 @@ public void setUp()
     {
         List<Product> prods = new ArrayList<Product>();
         shop = new Shop("Amazon", prods);
-        product = new Product("Name","Description", 20, "testPhoto".getBytes(), shop.getId());
+        product = new Product("Name","Description", 20, "testPhoto".getBytes(), shop.getId(), "https://dpa730eaqha29.cloudfront.net/mltnews/wp-content/uploads/2011/03/Events-in-Seattle-Starbucks-Coffee-Company.jpg");
```

Comment:
Lol does this mean that the images we use are just pulled/hosted somewhere else?

Reply:
yep. Easiest way to transfer images over.